### ISSUE TITLE
Error in the running the program

### ISSUE THREAD

**Issue by luisfegomezPE** [davidferguson/pibakery#189]
![imagen](https://user-images.githubusercontent.com/50106079/89744635-99d29e00-da74-11ea-96b6-bdba59c08fc3.png)

I have follow all the steps to install pibakery, but in the last step I recived this message. How can I solve this?

thanks.

**Comment by jmkebalo** [comment]
Hello,
I'm not mastering npm, but it seems the module semver was not install.
check on node_modules folder.
If any semserver, delete it, and run `npm install` again.

I had the issue with electron, it works for me

**Comment by tbandtg** [comment]
Yeah deleting it did not seem to work. It is in there and gets replaced but I get an error when running.

**Comment by jmkebalo** [comment]
I'm not sure to understand: If you delete it, where is it?
You should really remove manually the folder so it's not there anymore, and run the install again. It will download again, hopefully in a more proper way.

Can you share the error?

**Comment by tbandtg** [comment]
![image](https://user-images.githubusercontent.com/13041418/118129239-ea8f7280-b3c9-11eb-8a51-6c8776ea315f.png)
Here is the error I get, I think it is downloading a new version of semver that is no longer compatable.

**Comment by RPICtrl** [comment]
I Think it connects to pibakery.org and the site is Down.

**Comment by jmkebalo** [comment]
There is an issue while providing a version to the semver module.
I'm so not an expert on this part... I just put a quick (ugly) fix by hardcoding a version on the file node_modules/semver/server.js :
![image](https://user-images.githubusercontent.com/43848745/118264969-d70f0500-b4b8-11eb-9d53-19016b3b6c27.png)

It seems to work. But there is other issue to detect SD card... So I switch to Ansible to configure my rasp when the OS is installed.
Not a custom image but conf is deployed easily once the SSH keys are set.

**Comment by cllackey** [comment]
I'm using PiBakery v2 and this is a quick fix I found to the SD card error. In the main PiBakery project folder, go to resources/app/main.js and comment out line 104 which has this:
`electron.app.dock.hide()`
I was able to write to my SD card with no errors and nothing wonky happened.

_Edit: I will note that I couldn't boot my Pi after writing to the SD card but I have no way to confirm whether the issue was related to the afore mentioned change, if my img file was bad, if PiBakery v2 is faulty, etc._
